app/assets/stylesheets/master.scss

```diff
@@ -48,6 +48,10 @@ h3:after {
   background: image_url('bg.png');
 }
 
+.navbar-light .navbar-brand { 
+  font-family: "dubtronicsolid";
```

Comment:
I thought it was a shame we were using custom fonts but it's not anywhere on this page.  This isn't needed, but since you took the time to add custom fonts, I thought adding this could make sense.

Reply:
I tried adding them, but since they were not working I changed them to just plain old Helvetica.
thank you for fixing this